tests/acceptance/explore-test.js

```diff
@@ -0,0 +1,21 @@
+import { currentURL, click, visit } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { login } from 'open-event-frontend/tests/helpers/custom-helpers';
+
+module('Acceptance | explore', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /explore without login', async function(assert) {
+    await visit('/');
+    await click('.browseEvents');
```

Comment:
why this?
We can directly use 
```js
 await visit('/explore');
```

Reply:
I have edited the issue, we don't need the button to be clicked, sorry for the confusion.

Reply:
@CosmicCoder96 I've updated the PR but still the same error. Idk why travis is failing

Reply:
@shreyanshdwivedi They are failing because the code you wrote is working 😉 It's an acceptance test, and on visiting the explore route (test suite uses the heroku server deployment), a lot of requests are made for which the server returns a 404. These 404's are not being handled, hence the test does what it's meant to do in such scenario 😄Add a commit to this PR itself, and handle the errors.
![image](https://user-images.githubusercontent.com/17252805/50565928-01cde800-0d5a-11e9-9442-401b0a450168.png)

Reply:
Ok. Will fix it :sweat_smile:

Reply:
@CosmicCoder96 I think I can use some help here. I explored but did not get any reference of help :(

Reply:
@shreyanshdwivedi I will take a look at it, and get back :)